hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerInfo.java

```diff
@@ -133,8 +139,9 @@ public static ContainerInfo fromProtobuf(HddsProtos.ContainerInfoProto info) {
    * {@link ContainerID} object.
    */
   @Deprecated
```

Comment:
Maybe drop the deprecated flag? Looks like this API is here to stay.

Reply:
Yea, I think you could be correct. There is a need to get the numeric ID in a lot of places, so removing this is probably never going to happen.